openrtb_ext/bidders.go

```diff
@@ -604,6 +633,15 @@ func NewBidderParamsValidator(schemaDirectory string) (BidderParamValidator, err
 		schemaContents[BidderName(bidderName)] = string(fileBytes)
 	}
 
+	//set alias bidder params schema to its parent
+	for alias, parent := range aliasBidderToParentBidder {
+		parentSchema := schemas[parent]
+		parentSchemaContents := schemaContents[parent]
+
+		schemas[alias] = parentSchema
+		schemaContents[alias] = parentSchemaContents
+	}
```

Comment:
Nitpick: Consider pairing up the assignments like:

```
parentSchema := schemas[parent]
schemas[alias] = parentSchema

parentSchemaContents := schemaContents[parent]
schemaContents[alias] = parentSchemaContents
```

Reply:
https://github.com/prebid/prebid-server/pull/3048/commits/8e4cd4c9252d8b836b76b0833cb201c24d58aa3a